client/change.go

```diff
@@ -68,8 +68,9 @@ type Task struct {
 }
 
 type TaskProgress struct {
-	Done  int `json:"done"`
-	Total int `json:"total"`
+	Info  string `json:"info"`
```

Comment:
Can we please use "Label" here and everywhere else instead of Info? The "progress label" feels slightly more appropriate, and avoids more infos on that pipeline (we have taskInfo, snap.Info, etc).

Sorry, I realize this will be boring to rename.